client/test/findLangOrder.ts

```diff
@@ -155,145 +155,155 @@ test.serial('$lang should change the order when relevant', async (t) => {
     ],
   })
 
-  t.deepEqual(await client.get({
-    $language: 'en',
-    $id: 'match1',
-    children: {
-      id: true,
-      $list: {
-        $sort: { $field: 'title', $order: 'asc' },
-        $find: {
-          $traverse: 'descendants',
+  t.deepEqual(
+    await client.get({
+      $language: 'en',
+      $id: 'match1',
+      children: {
+        myId: { $field: 'id' },
+        $list: {
+          $sort: { $field: 'title', $order: 'asc' },
+          $find: {
+            $traverse: 'descendants',
+          },
         },
       },
-    },
-  }),
-  {
-    children: [
-      { id: 'team8' },
-      { id: 'team4' },
-      { id: 'team6' },
-      { id: 'team2' },
-      { id: 'team10' },
-      { id: 'team9' },
-      { id: 'team5' },
-      { id: 'team1' },
-      { id: 'team7' },
-      { id: 'team3' },
-    ]
-  })
+    }),
```

Comment:
The order is different between darwin and linux, I'll try to make a version for both.